mono/mini/mini-ppc.h

```diff
@@ -323,4 +343,32 @@ void mono_ppc_patch_plt_entry (guint8 *code, gpointer *got, mgreg_t *regs, guint
 
 void mono_ppc_set_func_into_sigctx (void *sigctx, void *func);
 
+
+// Debugging macros for ELF ABI v2
+#define DEBUG_ELFABIV2
```

Comment:
This probably shouldn't be on by default?

Also, perhaps these debug macros should be in `mini-ppc.c` instead of the header?